src/plugins/foxPage/components/AssetActions.tsx

```diff
@@ -9,39 +11,78 @@ import {
   Tabs,
   Text as CText,
 } from '@chakra-ui/react'
+import { AssetId } from '@shapeshiftoss/caip'
+import { foxyAddresses } from '@shapeshiftoss/investor-foxy'
+import { FoxyPath } from 'features/defi/providers/foxy/components/FoxyManager/FoxyManager'
+import qs from 'qs'
 import { useTranslate } from 'react-polyglot'
+import { useHistory, useLocation } from 'react-router'
 import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
 import { Text } from 'components/Text/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useModal } from 'hooks/useModal/useModal'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { useGetAssetDescriptionQuery } from 'state/slices/assetsSlice/assetsSlice'
+import { trimWithEndEllipsis } from 'state/slices/portfolioSlice/utils'
+import { selectAssetById } from 'state/slices/selectors'
+import { selectAccountIdsByAssetId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FoxAssetId, TrimmedDescriptionLength } from '../constants'
 
 type FoxTabProps = {
-  assetIcon: string
-  assetSymbol: string
-  description: string
-  buyCTA: string
-  sellCTA: string
-  onReceiveClick: () => void
-  onBuyClick: () => void
+  assetId: AssetId
 }
 
-export const AssetActions = ({
-  assetIcon,
-  assetSymbol,
-  description,
-  buyCTA,
-  sellCTA,
-  onReceiveClick,
-  onBuyClick,
-}: FoxTabProps) => {
+const GetFoxyModalRoute = FoxyPath.Overview
+const GetFoxCoinbaseExternalUrl = `https://www.coinbase.com/price/fox-token`
```

Comment:
```suggestion
const GetFoxyModalRoute = FoxyPath.Overview
const GetFoxCoinbaseExternalUrl = `https://www.coinbase.com/price/fox-token`
```

these aren't really getters... they should probably also be uppercased as consts?

Reply:
Uppercased one const and removed the other one as we could directly use `FoxyPath.Overview`